configure.ac

```diff
@@ -6215,6 +6214,12 @@ PY_STDLIB_MOD([_testmultiphase], [test "$TEST_MODULES" = yes])
 PY_STDLIB_MOD([_xxtestfuzz], [test "$TEST_MODULES" = yes])
 PY_STDLIB_MOD([_ctypes_test], [test "$TEST_MODULES" = yes], [], [], [-lm])
 
+dnl Template modules.
+PY_STDLIB_MOD_SIMPLE([_xxsubinterpreters])
```

Comment:
``_xxsubinterpreters`` is not a template. It's an experimental module with bindings to subinterpreter API.

Reply:
Right, assumption hits me again.